### PR TITLE
Fix visibility issue of dropdowns in dashboard

### LINKED ISSUE #1779: Filter combobox display bug
In list filter, we can't see choices that are after the filter container.

See screenshot:

saleor_filter_bug

### PR DESCRIPTION
**Pull request by NyanKiyoshi**
Fixes #1779.

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Python code quality checks pass: `pycodestyle`, `pydocstyle`, `pylint`.
1. [x] JavaScript code quality checks pass: `eslint`.

### CODE REVIEW

File: saleor/static/dashboard/scss/components/_card-collapse.scss [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .collapse {
-  overflow: hidden;
   max-height: 1000px;
```
Comment (dominik-zeglen) [inline saleor/static/dashboard/scss/components/_card-collapse.scss]: It's not as simple as that. Please notice that `overflow: hidden` is used to trim content in responsive mode, therefore this property shouldn't be simply removed. I'd wrap it in a `@media (max-width: 992px)` to completely shut it down on desktop-sized displays and try disabling it while filter bar is expanded, using `transition-delay: $transition-duration`. Good luck :)

Comment (NyanKiyoshi) [reply saleor/static/dashboard/scss/components/_card-collapse.scss]: I gonna try to have a look into that! I saw your commit before doing this PR using git blame but didn't find any breaking. I'm gonna look what I can do.

Comment (NyanKiyoshi) [reply saleor/static/dashboard/scss/components/_card-collapse.scss]: After a look, I see the issue, I didn't see it when trying on desktop yesterday. I'm suggesting to move the overflow hidden to `.collapse.collapsed`:
```css
.collapse.collapsed {
    overflow: hidden;
}
```

I'll be waiting to hear back from you.

Comment (dominik-zeglen) [reply saleor/static/dashboard/scss/components/_card-collapse.scss]: Okay, it looks cool - I thought that maybe filter bar's content would be visible while animation was running, but it works okay.

**Pull request merged**